spec/unit/installer/xcode/pods_project_generator/pod_target_installer_spec.rb

```diff
@@ -757,6 +767,34 @@ class PodsProjectGenerator
               @pod_target.dummy_source_path.read.should.include?('@interface PodsDummy_BananaLib')
             end
 
+            it 'creates an info.plist file when frameworks are required' do
+              @pod_target.stubs(:requires_frameworks?).returns(true)
+              @installer.install!
+              group = @project['Pods/BananaLib/Support Files']
+              group.children.map(&:display_name).sort.should == [
+                'BananaLib-Info.plist',
+                'BananaLib-dummy.m',
+                'BananaLib-prefix.pch',
+                'BananaLib.modulemap',
+                'BananaLib.xcconfig',
+              ]
```

Comment:
I couldn't find an existing unit test that covered this - if there is one I'll remove this